docs/changes.rst

```diff
@@ -305,6 +305,10 @@ and maybe just were not noticed.
 
 Compatibility notes:
 
+- matching of path patterns has been aligned with borg storing relative paths.
+  Borg archives file paths without leading slashes. Previously, include/exclude
+  patterns could contain leading slashes. You should check your patterns and
+  remove leading slashes.
```

Comment:
good, I'll copy that to above 1.2.1 entry when releasing that, so people will hopefully read it.